collect/collect.go

```diff
@@ -483,6 +498,7 @@ func (i *InMemCollector) processSpan(sp *types.Span) {
 		trace.SendBy = time.Now().Add(timeout)
 		trace.RootSpan = sp
 	}
+	i.Metrics.Increment("span_processed")
```

Comment:
Because of the deferred func will we be double-incrementing `span_processed`.

Reply:
Yup! Good catch.